DPGAnalysis/MuonSysAging/test/produceAgingObject_cfg.py

```diff
@@ -0,0 +1,161 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("CREATESQLITE")
+process.load("CondCore.CondDB.CondDB_cfi")
+process.load('Configuration.StandardSequences.GeometryDB_cff')
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+from Configuration.AlCa.autoCond import autoCond
+process.GlobalTag.globaltag = autoCond['run2_design']
+
+process.CondDB.connect = 'sqlite_file:MuonSystemAging.db'
+
+process.source = cms.Source("EmptyIOVSource",
+    lastValue = cms.uint64(1),
+    timetype = cms.string('runnumber'),
+    firstValue = cms.uint64(1),
+    interval = cms.uint64(1)
+)
+
+process.PoolDBOutputService = cms.Service("PoolDBOutputService",
+    process.CondDB,
+    timetype = cms.untracked.string('runnumber'),
+    toPut = cms.VPSet(cms.PSet(
+        record = cms.string('MuonSystemAgingRcd'),
+        tag = cms.string('MuonSystemAging_test')
+    ))
+)
+
+process.produceAgingObject = cms.EDAnalyzer("ProduceAgingObject",
```

Comment:
To give an example, in the DT case at least, aging is well connected with the flux a chamber is exposed. 
Hence there are symmetries in the production of aging scenarios, that would make the use of regexes better than multiple cut-and-paste.

Afterall this is just the tool run by the user to generate andaging DB object, how stringent the reason justifying the choice should be?